cypress/TestCases/FireboltCertification/FederatedData.feature

```diff
@@ -0,0 +1,115 @@
+            | With entity episodeNumber                 | get entity details with episodeNumber   | expected entity episodeNumber                |
+            | With entity SeasonNumber                  | get entity details with SeasonNumber    | expected entity SeasonNumber                 |
+            | With entity Synopsis                      | get entity details with Synopsis        | expected entity Synopsis                     |
+            | With entity ReleaseDate                   | get entity details with ReleaseDate     | expected entity ReleaseDate                  |
+            | With entityInfo all values                | get entity details with all values      | expected entity details with all values      |
+            | Without entity type                       | get entity details without entity type  | expected entity details without entity type  |
+            | With entity type                          | get entity details with entity type     | expected entity details with entity type     |
+            | Without program type but with entity type | get entity details without program type | expected entity details without program type |
+
+    @FederatedData @coreSDK @sdk @transport
+    Scenario Outline: Discovery_content.requestDetails - Negative Scenario: <Scenario> expecting error
+        When 1st party app invokes the 'Firebolt' API to '<API_Key>'
+        Then 'Firebolt' platform responds to '1st party app' with '<Validation_Key>'
+
+        Examples:
+            | Scenario           | API_Key                                    | Validation_Key                                |
+            | Without Expires    | get entity details without expires         | custom error for content requestDetails       |
+            | Invalid EntityId   | get entity details with invalid entityId   | invalid parameters for content requestDetails |
+            | Invalid Parameters | get entity details with invalid parameters | invalid parameters for content requestDetails |
+
+    @FederatedData @coreSDK @sdk @transport
+    Scenario: Discovery_content.requestPurchases - Positive Scenario: Purchases
+        When 1st party app invokes the 'Firebolt' API to 'get list of purchases'
+        Then 'Firebolt' platform responds to '1st party app' with 'expected purchases'
+
+    @FederatedData @coreSDK @sdk @transport
+    Scenario: Discovery_content.requestPurchases - Negative Scenario: Purchases with integer
+        When 1st party app invokes the 'Firebolt' API to 'get list of purchases with integer'
+        Then 'Firebolt' platform responds to '1st party app' with 'invalid parameters for purchases'
+
+    # for Discovery.details and Discovery.purchases added TCs as per new openRPC.
+    @FederatedData @coreSDK @sdk @transport
+    Scenario Outline: Discovery.details - Positive Scenario: <Scenario>
+        When '3rd party app' invokes the 'Firebolt' API to '<API_Key>'
```

Comment:
got an info from kevin that the flow of discovery.details and discovery.purchases is different from discovery.entityInfo and discovery.purchasedContent, not sure whether normal testcases can work

Reply:
not sure about the flow so added normal testcases as per openRPC.